Author: Andrew Lee (李健秋) <ajqlee@debian.org>
Description: Change default terminal and browser.

Index: lxpanel-0.7.2/data/default/panels/panel.in
===================================================================
--- lxpanel-0.7.2.orig/data/default/panels/panel.in	2015-01-31 15:49:37.585637050 +0100
+++ lxpanel-0.7.2/data/default/panels/panel.in	2015-01-31 15:49:37.577637274 +0100
@@ -55,10 +55,10 @@
             id=pcmanfm.desktop
         }
         Button {
-            id=gnome-terminal.desktop
+            id=lxde-x-terminal-emulator.desktop
         }
         Button {
-            id=firefox.desktop
+            id=lxde-x-www-browser.desktop
         }
     }
 }
